framework/vkdf-buffer.cpp

```diff
@@ -90,7 +100,7 @@ vkdf_buffer_map_and_fill_elements(VkdfContext *ctx,
       src_offset += src_stride;
    }
 
-   vkdf_memory_unmap(ctx, buf.mem, buf.mem_props, offset, size);
+   vkdf_memory_unmap(ctx, buf.mem, buf.mem_props, offset, mapping_size);
 }
 
 void
```

Comment:
One other comment: change the shortlog to this:

"framework/buffer: align buffer size for map/unmap to honor memory requirements"